src/reporter/reporter.py

```diff
@@ -147,15 +168,26 @@ def notify():
         fiware_sp = request.headers.get('fiware-servicepath', None)
     else:
         fiware_sp = None
-
-    # Send valid entities to translator
-    with translator_for(fiware_s) as trans:
-        trans.insert(payload, fiware_s, fiware_sp)
-
-    msg = 'Notification successfully processed'
```

Comment:
I thought we should get this code block back in and zap the `for` loop below? I reckon there's something I'm missing here, oh deary deary :-)

Reply:
@c0c0n3 , when we receive payload the it will validate it below the code.

Reply:
coolio, thanks!